src/components/Dialogs/FittingDialog/FittingDialogComponent.scss

```diff
@@ -48,6 +48,32 @@
         margin-bottom: 0;
     }
 
+    .output-result-button {
+        position: absolute;
+        bottom: 25px;
+        right: 25px;
+        margin: 0 0 0 0;
+        transition: opacity 400ms;
```

Comment:
Looks like the transition is not working.

Reply:
It's working now with the modification shown below:
![image](https://user-images.githubusercontent.com/106953609/193226479-72eb96ef-2748-46a0-8a55-9d7ed1bbdea0.png)